{"name": "困人事件标记误报", "status": "passed", "attachments": [{"name": "log", "source": "c28c4961-ff8b-47ec-b5c3-f58516bd2967-attachment.txt", "type": "text/plain"}], "start": 1690185382082, "stop": 1690185382158, "uuid": "c49b4e80-af88-468b-8011-6bcb15d0dfa1", "historyId": "ab566412ce2c9ed8e210215e4ae8a5dd", "testCaseId": "16b1c3c86243e1b02c591065644e1507", "fullName": "opsPC.test_emergency#test_update_deal_status_false_alarm_001", "labels": [{"name": "story", "value": "应急处置"}, {"name": "feature", "value": "运维管理平台"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_emergency"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "18100-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_emergency"}]}